.github/workflows/ci.yaml

```diff
@@ -54,10 +54,13 @@ jobs:
         python-version: ${{ matrix.python-version }}
         allow-prereleases: true
 
+    - name: Test build
+      run: pipx run --python=python build .
```

Comment:
I see that `build` is called before the deps are installed.

Is that ok ?

Reply:
pipx manages the deps